config/kubernetes/ug-big-data/OWNERS

```diff
@@ -1,8 +1,8 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 reviewers:
-  - sig-big-data-leads
+  - ug-big-data-leads
 approvers:
-  - sig-big-data-leads
+  - ug-big-data-leads
 labels:
-  - sig/big-data
+  - usergroup/big-data
```

Comment:
This label doesn't exist today..I'll create a PR to add it once https://github.com/kubernetes/community/pull/3304 has merged.

Reply:
The label also needs to be `ug/big-data` to stay consistent with other label conventions.